src/py42/clients/alerts.py

```diff
@@ -61,4 +71,34 @@ def reopen(self, alert_ids, reason=None):
         Returns:
             :class:`py42.response.Py42Response`
         """
-        return self._alert_service.reopen(alert_ids, reason=reason)
+        return self._alert_service.update_state(
+            AlertState.OPEN, alert_ids, reason=reason
+        )
+
+    def pending(self, alert_ids, reason=None):
```

Comment:
I think instead of adding a new method for each of the other states, we should just have a catch-all like `up=date_state()`.  `pending()` is not a function-y sounding name and If other states come along, this way we already can support them.  I feel like we need other devs opinions on this design decision though:  @alanag13 , @timabrmsn , @kiran-chaudhary

Reply:
Yea, we can redesign it.  I did think about it but then thought that would be considered a breaking change, also thought  of `to_pending` but later on went ahead with `pending`.

Reply:
It is not a breaking change as long as we leave the original `resolve` and `reopen` methods

Reply:
Changed.